com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -440,8 +440,9 @@ else if (fontDesc.getOutputFormat() == FontTextureFormat.TYPE_DISTANCE_FIELD) {
         if (fontDesc.getOutputFormat() == FontTextureFormat.TYPE_BITMAP &&
             inputFormat == InputFontFormat.FORMAT_TRUETYPE) {
 
-            // If font has outline, we need all three channels
-            if (fontDesc.getOutlineWidth() > 0.0f && this.fontDesc.getOutlineAlpha() > 0.0f) {
+            // If font has outline or shadow, we need all three channels
+            if ((fontDesc.getOutlineWidth() > 0.0f && this.fontDesc.getOutlineAlpha() > 0.0f)
+            || (fontDesc.getShadowAlpha() > 0.0f)) {
```

Comment:
Why does only "outline" need to check the width?
They (shadow & outline), seem similar in behaviour, so should they not have similar if-statements?

Reply:
> Why does only "outline" need to check the width?

There's no need to draw the outline if the width of the outline is 0. And there's also no need to draw the outline if the alpha is 0.

For the shadow on the other hand, we only check alpha. Blur can still be 0.

Reply:
Ah, yes, I incorrectly thought it has a shadow width